randovania/games/am2r/json_data/The Nest.txt

```diff
@@ -529,7 +529,8 @@ Extra - map_name: rm_a6b02
   * Open Passage to Depths Shinespark Cave West/Dock to Depths Shinespark Cave (Top Left)
   > Dock to Depths Hub
       Any of the following:
-          Hi-Jump Boots or Space Jump or Can Use Spider Ball
+          Space Jump or Can Use Spider Ball
+          Hi-Jump Boots and Walljump (Beginner)
```

Comment:
is WJ really needed?

Reply:
yes, with HJ you only make it to the wall below the left side platform so you need at least 1 WJ.